packages/components/src/toolbar/style.scss

```diff
@@ -67,23 +67,25 @@ div.components-toolbar {
 
 	& > svg {
 		padding: 5px;
-		box-sizing: content-box;
 		border-radius: $radius-round-rectangle;
+		height: 30px;
+		width: 30px;
 	}
 
 	// Subscript for numbered icon buttons, like headings
 	&[data-subscript] svg {
-		padding: 4px 8px 4px 0;
+		padding: 5px 10px 5px 0;
 	}
 
 	&[data-subscript]::after {
 		content: attr(data-subscript);
 		font-family: $default-font;
 		font-size: $default-font-size;
 		font-weight: 600;
+		line-height: 12px;
```

Comment:
Is this based on the `font-size` we assign? And if so, should it be based on `$default-font-size`, or a multiplier of said value?

Reply:
It's kind of random to make it consistent across the browsers.

Reply:
😱

Reply:
The placement of the subscribe is very hard to get right :). Even with this, I'm not certain about it so open to suggestions

Reply:
By random, I meant any value can work, we just need to adjust the "bottom" accordingly.